src/components/apiReference/Object.tsx

```diff
@@ -8,57 +8,51 @@ import { TableOfContentsItem } from "../TableOfContents/TableOfContents";
 import "./reference.scss";
 
 interface ObjectProperty {
-    name: string,
-    description: string
+    name: string;
+    description: string;
 }
 
 interface ObjectData {
-    description: string,
-    properties: ObjectProperty[]
-};
+    description: string;
+    properties: ObjectProperty[];
+}
 
 export default function Object(): JSX.Element {
     const { objectId = "" } = useParams();
     const storedObjectData: ObjectData = JSON.parse(sessionStorage[objectId]);
     const objectDescription: string = storedObjectData.description;
     const properties: ObjectProperty[] = storedObjectData.properties;
 
-    const introductionParagraph: ReactNode = (
-        <div className="paragraph">
-            {RenderStringWithUrl(objectDescription)}
-        </div>
-    );
+    const introductionParagraph: ReactNode = <div className="paragraph">{RenderStringWithUrl(objectId, objectDescription)}</div>;
 
     const propertyParagaphs: ReactNode = (
         <div className="paragraph">
-            {properties.map((property: any) =>
-                <div>
+            {properties.map((property: ObjectProperty) => (
+                <div key={property.name}>
                     <h3>{property.name}</h3>
-                    <p>{RenderStringWithUrl(property.description)}</p>
+                    {RenderStringWithUrl(property.name, property.description)}
                 </div>
-            )}
+            ))}
         </div>
     );
 
     const pageTitle: PageTitleProps = {
-        "title": objectId,
-        "breadCrumbItems": ["MYG", "API Reference", "Objects", objectId]
+        title: objectId,
+        breadCrumbItems: ["MYG", "API Reference", "Objects", objectId]
     };
 
     const pageSections: TableOfContentsItem[] = [
         {
-            "elementId": "introduction",
-            "summary": "Introduction",
-            "details": introductionParagraph
-        }, 
+            elementId: "introduction",
+            summary: "Introduction",
+            details: introductionParagraph
+        },
         {
-            "elementId": "properties",
-            "summary": "Properties",
-            "details": propertyParagaphs
+            elementId: "properties",
+            summary: "Properties",
+            details: propertyParagaphs
```

Comment:
Typo that we missed in the previous branch

```suggestion
            details: propertyParagraphs
```